test/helper.py

```diff
@@ -52,7 +52,7 @@
 localhost = testinfra.get_host('local://')
 
 BUILD_BUNDLE_SCRIPT = (
-    PROJECT_PATH
+    (PROJECT_PATH if PROJECT_PATH else PRVSNR_ROOT_DIR)
```

Comment:
I think we can simplify this to that shorter form
```python
BUILD_BUNDLE_SCRIPT = PROJECT_PATH or PRVSNR_ROOT_DIR
```

Reply:
updated